src/main/java/frc/robot/classes/TimeOfFlightSensor.java

```diff
@@ -0,0 +1,234 @@
+package frc.robot.classes;
+
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.locks.ReentrantLock;
+
+import edu.wpi.first.hal.HAL;
+import edu.wpi.first.hal.SerialPortJNI;
+import edu.wpi.first.hal.FRCNetComm.tResourceType;
+import edu.wpi.first.wpilibj.Timer;
+
+public class TimeOfFlightSensor implements AutoCloseable {
+  public static class RawDistance {
```

Comment:
Raw COlor was necessary because you could have uninterpreted numbers for color, or adjusted numbers.

Do we need this to be a class?  Or can we just return the integer distance?
Code would be cleaner if we make this just an integer and not a class